ecf/scripts/global_ens/prep/jevs_global_ens_chem_grid2obs_prep.ecf

```diff
@@ -0,0 +1,94 @@
+#PBS -N jevs_global_ens_chem_g2o_prep
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=01:00:00
+#PBS -l select=ncpus=1:mem=8GB
+#PBS -l debug=true
+#PBS -o g2o_out.t00z
+#PBS -e g20_err.t00z
+#PBS -V
+
+##%include <head.h>
+##%include <envir-p1.h>
+
+############################################################
+# Load modules
+############################################################
+set -x
+
+export model=evs
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
+source $HOMEevs/versions/run.ver
+
+module reset
+export HPC_OPT=/apps/ops/para/libs
+module use /apps/ops/para/libs/modulefiles/compiler/intel/${intel_ver}
+module use /apps/dev/modulefiles/
+module load ve/evs/${ve_evs_ver}
+module load cray-mpich/${craympich_ver}
+module load cray-pals/${craypals_ver}
+module load libjpeg/${libjpeg_ver}
+module load libpng/${libpng_ver}
+module load zlib/${zlib_ver}
+module load jasper/${jasper_ver}
+module load cfp/${cfp_ver}
+module load gsl/${gsl_ver}
+module load met/${met_ver}
+module load metplus/${metplus_ver}
+module load prod_util/${prod_util_ver}
+module load prod_envir/${prod_envir_ver}
+
+module list
+
+export MET_PLUS_PATH="/apps/ops/para/libs/intel/${intel_ver}/metplus/${metplus_ver}"
+export MET_PATH="/apps/ops/para/libs/intel/${intel_ver}/met/${met_ver}"
+export MET_CONFIG="${MET_PLUS_PATH}/parm/met_config"
+export PYTHONPATH=$HOMEevs/ush/$COMPONENT:$PYTHONPATH
+
+export cyc=00
+echo $cyc
+export KEEPDATA=YES
+export NET=evs
+export STEP=prep
+export COMPONENT=global_ens
+export RUN=chem
+export VERIF_CASE=grid2obs
+export MODELNAME=gefs
+
+######################################
+### Correct MET/METplus roots (Aug 2022)
+########################################
+export MET_bin_exec=bin
+
+export VDATE=$(date --date="2 days ago" +%Y%m%d)
+
+export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/${evs_ver}
+export COMINobs=/lfs/h1/ops/dev/dcom/${VDATE}
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/${evs_ver}
+export COMOUTprep=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/${evs_ver}/$STEP/$COMPONENT
+export DATA=/lfs/h2/emc/ptmp/$USER/$NET/${evs_ver}
+mkdir -p DATA
+export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix
+export USHevs=$HOMEevs/ush/$COMPONENT
+export CONFIGevs=$HOMEevs/parm/metplus_config/$COMPONENT
+export PARMevs=$HOMEevs/parm/metplus_config
+
+## developers directories
+
+export cycle=t${cyc}z
+
+export VDATE=$PDYm2
+
+############################################################
+## CALL executable job script here
+#############################################################
+$HOMEevs/jobs/$COMPONENT/$STEP/JEVS_GLOBAL_ENS_CHEM_GRID2OBS_PREP
+
+#%include <tail.h>
+#%manual
+#######################################################################
+# Purpose: This does the prep work for the global_ens GEFS-Chem model
+#######################################################################
+#%end
```

Comment:
@Ho-ChunHuang-NOAA Appreciate your attention to detail in adding in the ecf scripts.  However, it would probably be a good idea to use an existing ecf script as a guide as to how yours should look like.  I recommend taking a look at some existing ecf scripts.

In the meantime, ecf scripts are not needed to run this PR.  But we'll definitely need to update these ecf files to be files that are acceptable for NCO to run in ecflow.  

@AliciaBentley-NOAA @malloryprow for awareness.

Reply:
Perry, please be more specific on the request for revision.  I already referenced the existing ecf script as much as I can.  I may miss something and I need your experience to help me on what to change to meet the standard. 

For your comment do you want me to remove the comment at the end of ecf script? is there other revision I need?  Thank you.

Reply:
Yes I was just reviewing the ecf script. I would like the ecf scripts to be updated in this PR, so @Ho-ChunHuang-NOAA please look at other ecf scripts and update accordingly. We are here to help if you need. Please make sure the resources in the ecf scripts match those in those corresponding dev/driver/scripts too.

Reply:
@Ho-ChunHuang-NOAA The entire structure of the ecf script should look like the other ecf scripts.  Please look at some of the other ecf scripts that are currently in the ecf directory.  For example, look at this file:

https://github.com/Ho-ChunHuang-NOAA/EVS/blob/feature/GEFSchem/ecf/scripts/prep/global_ens/jevs_global_ens_atmos_prep.ecf

Your ecf script should match this structure.

Reply:
Hi, Perry:

The ~/ecf/scripts/global_ens should be removed.  Please check the /ecf/scripts/prep/global_ens, /ecf/scripts/stats/global_ens, and /ecf/scripts/plots/global_ens.

Sorry for the error.

Reply:
Oh!  I didn't know that this was from an extraneous directory.  

That said - we do need to include ecf scripts with this submission.  We can help with that.  

Please remove this directory from this PR.

Reply:
It is done for the removal.  I already created the ecf scripts in corresponding directory, please review them.  

e.g., https://github.com/Ho-ChunHuang-NOAA/EVS/blob/feature/GEFSchem/ecf/scripts/prep/global_ens/jevs_global_ens_chem_grid2obs_prep.ecf

Thanks.

Reply:
The new ecf scripts look fantastic @Ho-ChunHuang-NOAA!! Thank you do making the changes!!